action.yml

```diff
@@ -199,6 +199,108 @@ runs:
           cd /tmp/skaffold
           skaffold config set collect-metrics false
           skaffold deploy
+
+          if [ $? -eq 0 ]; then
+            skaffold render > rendered.yaml
+
+            # Check if the command failed
+            if [ ${command_failed:-0} -eq 1 ]; then
+              echo "WARN: failed to save artifact information: '$response'"
+            fi
+
+            container_images=""
+            init_container_images=""
+        
+            if yq -e '.spec.template.spec.containers[]?.image' "rendered.yaml" > /dev/null 2>&1; then
+              # Extract the unique container images from the rendered chart
+              container_images=$(yq '.spec.template.spec.containers[]?.image' "rendered.yaml" | sort -u)
+            fi
+        
+            if yq -e '.spec.template.spec.initContainers[]?.image' "rendered.yaml" > /dev/null 2>&1; then
+              # Extract unique init container images
+              init_container_images=$(yq '.spec.template.spec.initContainers[]?.image' "rendered.yaml" | sort -u)
+            fi
+            
+            # Combine both lists of images, ensuring uniqueness
+            all_images=$(echo -e "$container_images\n$init_container_images" | sort -u)
+
+            url=""
+            sha256_digest=""
+            chart_name=""
+            chart_version=""
+
+            if [[ "$CHART_LOCATION" =~ ^oci:// ]]; then
+              # OCI image format
+              helm_pull_output=$(helm pull "$CHART_LOCATION" 2>&1)
+              sha256_digest=$(echo "$helm_pull_output" | grep 'Digest:' | sed 's/Digest: //' || echo "" )
+              chart_name=$(echo "$helm_pull_output" | grep 'Pulled:' | sed 's/Pulled: //' | sed 's/:.*//' || echo "")
```

Comment:
Lets use the chart name as the name of the artifact from the `Chart.yaml` for both local as well as remote chart.